### ISSUE TITLE
Adding Vietnamese

### ISSUE THREAD

**Issue by crystaltype** [TiroTypeworks/Slabo#9]
Hi,

I see that this type doesn't have Vietnamese yet.

I'm willing to contribute the complete Vietnamese character set directly, by sending you a 'vietnamese only' glyphapp file for you to review and integrate.

Or, you can make changes and then push your work to master and I will give further feedback :)

Please respond ASAP. Thanks!

P/S: I'm a Vietnamese native speaker and a self-study graphic designer/type designer. I have been working with fonts for over 5 years! I'm here to (hopefully) make google fonts a better place for people using Vietnamese! :) Thanks!

cc': @davelab6

**Comment by tiroj** [comment]
Yes, I'd be happy to integrate Vietnamese support into Slabo, so long as it maintains the pixel grid fit approach of the other glyphs. The preferred approach would be an initial set of size-independent glyphs added to Slabo Xpx font, and then updates derived from those outlines for the size-specific 13px and 27px fonts. [I have, by the way, a partly completed 22px version, which I hope to finish up during the summer.]

I shall be away for most of April, so would not be able to do any integration work on my end until after I return. I also expect to be very busy in May and June, so am not sure how quickly I'd be able to do that work.

**Comment by crystaltype** [comment]
I think we won't have any problems, since Vietnamese use Latin alphabet after all. I just have to use your existing design and add more Vietnamese diacritics, and we're done!

I'll use your ufo sources and save them as glyphsapp sources, and make a pull request for you, or I can send them directly by email. Which do you prefer?

Don't worry about it, I think you can look at it whenever you have the time. :) Thanks for replying!

**Comment by tiroj** [comment]
Some notes that may help:

The different px fonts have different UPM (units per em) values, which allowed me to precisely divide the UPM value by the number of target pixels to define a design grid.

Slabo 27px:
UPM = 810
Grid = 30x30 font units

Slabo 13px:
UPM = 780
Grid = 60x60 font units

If you set up corresponding grids in Glyphs, you'll see how the outlines and, especially, heights fit to the grid.

Please make precomposed Vietnamese mark combinations as zero-width mark ligatures, using the same offset as in the other marks in the font: this will allow us to use the same component positions in the diacritic characters.

Please send the files by email, and I'll update all the sources on my end before refreshing the repository.

Thanks for this. I look forward to seeing the results.
